public/electron.js

```diff
@@ -82,15 +83,31 @@ function initialize(queues) {
   )
 
   mainWindow.webContents.on('did-finish-load', () => mainWindow.show())
-  // mainWindow.webContents.openDevTools()
+
+  // open devtools if DEV_TOOLS is set to 1
+  if (parseInt(process.env.DEV_TOOLS)) {
+    mainWindow.webContents.openDevTools()
+  }
+
   mainWindow.on('closed', () => {
     mainWindow = null
   })
 }
 
 app.on('ready', async () => {
 
-  globalShortcut.register('CommandOrControl+Q', () => app.quit())
+  globalShortcut.register('CommandOrControl+Q', () => {
+
+    // prompt user before closing the application if there are active jobs
+    if (Object.keys(queues['queues']['video transcoding']['childPool'].retained).length != 0) {
```

Comment:
Are you sure this checks if _any_ active jobs are still running? Seems to be looking only at video transcoding jobs?

Reply:
Would probably be more precise to check if the length is positive.